antora-ui-camel/src/css/footer.css

```diff
@@ -182,59 +181,45 @@ input[type="checkbox"] {
     width: auto;
   }
 
-  footer .footer .context a {
-    font-size: 13.5px;
-  }
-
-  footer .footer .context a::after {
-    width: 5px;
-    height: 5px;
-  }
-
   footer .footer .footer-icons .brand-icon {
     height: 1.5rem;
   }
 
-  footer .footer .show-menu {
-    float: right;
-    height: 40px;
-    margin-top: -2.5rem;
-  }
-
-  footer .footer .hide-menu {
-    float: right;
-    height: 35px;
-    margin-top: -2.5rem;
+  footer .footer input[type="checkbox"] {
+    display: inline;
+    position: relative;
+    top: 2rem;
+    left: 45vw;
+    width: 1.75rem;
+    height: 1.75rem;
+    transition: 0.2s;
   }
 
-  footer .footer .footer-menu {
-    margin-top: 1rem;
+  footer .footer input[type="checkbox"]:checked {
+    transform: rotate(45deg);
   }
 
-  input[type="checkbox"] + label img.show-menu {
+  footer .footer input[type="checkbox"]::before {
+    background-color: var(--footer-background);
+    content: '\FF0B';
+    position: absolute;
     display: block;
+    height: 1.75rem;
+    width: 1.75rem;
+    line-height: 1.75rem;
+    font-size: 1.75rem;
+    transition: 0.2s;
   }
 
-  input[type="checkbox"] ~ label img.hide-menu,
-  input[type="checkbox"] ~ div.footer-menu {
-    display: none;
-  }
-
-  input[type="checkbox"]:checked + label img.show-menu {
-    display: none;
-  }
-
-  input[type="checkbox"]:checked ~ label img.hide-menu,
-  input[type="checkbox"]:checked ~ div.footer-menu {
-    display: block;
+  footer .footer input[type="checkbox"]:checked::before {
+    transform: rotate(45deg);
```

Comment:
`transform: rotate(90deg);` 
This is 'cause even on clicking it rotates to a plus sign instead of a cross. To make it cross, the following code achieves it.

Reply:
This is caused by both `input` and `::before` being rotated by 45deg. I've gone with a different implementation in the latest changes.